Server_Side_TLS.mediawiki

```diff
@@ -36,16 +36,18 @@ The ordering of a ciphersuite is very important because it decides which algorit
 For services that don't need backward compatibility, the parameters below provide a higher level of security. This configuration is compatible with Firefox 27, Chrome 30, IE 11 on Windows 7, Edge, Opera 17, Safari 9, Android 5.0, and Java 8.
 
 * Ciphersuites: '''ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256'''
-* Versions: '''TLSv1.2'''
-* TLS curves: '''prime256v1, secp384r1, secp521r1'''
-* Certificate type: '''ECDSA'''
-* Certificate curve: '''prime256v1, secp384r1, secp521r1'''
-* Certificate signature: '''sha256WithRSAEncryption, ecdsa-with-SHA256, ecdsa-with-SHA384, ecdsa-with-SHA512'''
-* RSA key size: '''2048''' (if not ecdsa)
-* DH Parameter size: '''None''' (disabled entirely)
-* ECDH Parameter size: '''256'''
-* HSTS: '''max-age=15768000'''
-* Certificate switching: '''None'''
+* Versions: '''TLSv1.3, TLSv1.2'''
+* TLS curves: '''prime256v1, secp384r1, x25519, x448'''
+* Certificate type: '''ECDSA (recommended) or RSA'''
+* Certificate curve: '''prime256v1, secp384r1'''
+* Certificate signature: '''ecdsa-with-SHA256, ecdsa-with-SHA384, ecdsa-with-SHA512, sha256WithRSAEncryption, sha384WithRSAEncryption, sha512WithRSAEncryption'''
```

Comment:
I would nail the curves to the hashes TLS 1.3 style

Reply:
These here are right; you want these in the certificate, and I think that we can rely on TLS stacks to get the PSS thing right.  While text on PSS might help, it's probably not going to get the results you want (we might see PSS certificates, which won't work).

Reply:
> we might see PSS certificates, which won't work

they do work in OpenSSL and GnuTLS already; NSS is also slowly progressing towards that direction